airflow/models/taskinstance.py

```diff
@@ -456,7 +461,7 @@ class TaskInstance(Base, LoggingMixin):
     queued_dttm = Column(UtcDateTime)
     queued_by_job_id = Column(Integer)
     pid = Column(Integer)
-    executor_config = Column(PickleType(pickler=dill, comparator=_executor_config_comparator))
```

Comment:
We can get rid of `_executor_config_comparator` now, yeah?

Reply:
hmm good point.  so this new ExecutorConfigType will store things using our serializer on a go-forward.  however, for stuff already stored in the DB, we could still encounter that attribute error when comparing.  so, probably we should continue to use that as the comparator.... what do you think?

Reply:
did that -- see latest commit

Reply:
Yeah, that makes sense.